edgexfoundry/device-uart
# feat: Remove ZeroMQ MessageBus capability

closes #10







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run `make tidy` 
Verify the github.com/pebbe/zmq4 module is not in the go.mod file
run `make build` w/o ZMQ libs install on system
Verify builds successfully
run `make tests`
Verify tests all pass
run `make docker`
Verify builds docker image successfully

## New Dependency Instructions (If applicable)
